src/components/toast/Toast.jsx

```diff
@@ -1,12 +1,32 @@
-import { BiXLg } from "assets/icons/Icons";
+import { useAuth } from "context";
+import { useEffect } from "react";
+import "./toast.css";
+
+function Toast() {
+  const {
+    authState: {
+      toast: { type, msg, toastState },
+    },
+    authDispatch,
+  } = useAuth();
+
+  useEffect(() => {
+    const timeOut = setTimeout(() => {
+      authDispatch({
+        type: "SET_TOAST",
+        payload: {
+          state: false,
+          msg: "",
+          type: "",
+        },
+      });
+    }, 3000);
+    return () => clearTimeout(timeOut);
+  }, [authDispatch]);
```

Comment:
Suggestion:- Can remove the dependency array and just add an empty dependency array.

Reply:
vsCode gives a warning to include authDispatch